### PR TITLE
Fixed image hits and modernized packaging

### PR DESCRIPTION
**Pull request by induane**
- Replaced google backend with Yandex for much better results
- Added more animals & adjectives
- Updated packaging to use pyproject.toml, setuptools_scm and other tweaks

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,10 @@
+        DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
```
Comment (middlemarch) [inline LICENSE:1]: OMG our app is dropping F-bombs! :open_mouth: :rainbow:

File: src/dopameme/constants.py [hunk 2]
```diff
@@ -1,133 +1,143 @@
...
+    "piggy",
+    "pony",
+    "pupper",
+    "puppy",
+    "quokka",
+    "rabbit",
+    "raccoon",
+    "seal",
+    "sloth",
+    "snail",
```
Comment (middlemarch) [inline src/dopameme/constants.py:70]: I'll allow it :smile:

File: src/dopameme/constants.py [hunk 3]
```diff
@@ -1,133 +1,143 @@
...
-    'xxx',
-    'youporn',
-    'zillow',
+    "?show_error=true",
+    "amazon",
+    "etsy",
+    "goldporntube",
+    "google.com/search",
+    "item",
+    "pornhub",
```
Comment (middlemarch) [inline src/dopameme/constants.py:118]: I'm gonna hope this and the following are preemptive!

File: Makefile [hunk 4]
```diff
@@ -1,49 +1,60 @@
...
+sdist: build-reqs
+	$(IN_ENV) $(PYTHON) -m build --no-isolation --sdist
 
-wheel: build_reqs
-	$(IN_ENV) python setup.py bdist_wheel
+wheel: build-reqs
+	$(IN_ENV) $(PYTHON) -m build --no-isolation --wheel
 
-lint: pep8
+format-code:
```
Comment (middlemarch) [inline Makefile:43]: Love black formatting!

File: src/dopameme/constants.py (hunk 3 above)
Comment (induane) [reply src/dopameme/constants.py:118]: I added a few that came up one or twice in thousands of test runs. I want cute floofy bunnies not playboy bunnies!

File: Makefile (hunk 4 above)
Comment (induane) [reply Makefile:43]: I agree with black "most of the time" - and that's good enough for me. Yay for not having to think about it at all!

**Review by middlemarch — APPROVED** [review]
Welcome yandex to the cutiverse!

**Pull request merged**